Add Māori language support.

demiankatz opened the pull request:
This PR incorporates translations provided by @cmurdoch with auto-generated language name translations created using provided tooling. There is still significantly more translation work to do, but this provides a useful starting point.

demiankatz committed:
Add Māori translations, courtesy of Craig Murdoch.

demiankatz committed:
Add language translations.

demiankatz committed:
Augment language translations.

demiankatz committed:
Remove special character to prevent sorting problems.

demiankatz commented on languages/native.ini:
```diff
@@ -22,6 +22,7 @@ Hindi = "हिंदी"
 Irish = "Gaeilge"
 Italian = "Italiano"
 Japanese = "日本語"
+Maori = "Māori"
```
Note that I tried to use "Māori" everywhere instead of "Maori" but the special character caused some problems, so I ended up with "Maori" in config.ini, and this translation to apply the appropriate accent when displaying it in the UI. Not ideal, but I think it should be okay.

cmurdoch replied:
Thanks @demiankatz, to be correct we should probably use Maaori if it's impossible to add the special character.

demiankatz replied:
Good idea, I have adjusted this.

demiankatz committed:
Adjust spelling.

demiankatz committed:
Update native.ini

cmurdoch commented:
@demiankatz Just wondering about the first term pair in the file I sent you - Abstract = "Whakarāpopotonga". I don't see that in the new mi.ini file. Is this a result of the process you used, a one-off, or should I check thoroughly?

demiankatz committed:
Add missing line.

demiankatz commented:
Good eye, @cmurdoch. I'm not exactly sure how that happened, but I think it's likely to be a one-off. Just for full context, here's what I did:

1.) I deleted the "AUT only" lines from the bottom of the file, since they do not have equivalents in the other languages.
2.) I ran the language normalization tool to sort the remaining lines into standardized order.
3.) I deleted all of the comments from the top, but added a new one acknowledging you (I suspect at this point, I somehow selected one extra line and accidentally deleted the Abstract translation).
4.) I used the language file comparison tool to identify other unmatched translations in the file -- some of these were due to typos in the keys, others were due to keys becoming obsolete or changing. I fixed, updated or removed all of these as appropriate.

...and that's how we end up where we are now, with a new step 5 of adding back the line I apparently accidentally deleted. :-)

Thanks!

cmurdoch reviewed (approved):
Thanks @demiankatz all looks good to me.
